### ISSUE TITLE
Error with loss function

### ISSUE THREAD

**Issue by FPerezHernandez92** [ziyangyeh/iMeshSegNet#4]
Hello, sorry again for the inconvenience. I had to modify the following line to avoid the error I was getting in the attached file. Thank you very much for your work

https://github.com/ziyangyeh/iMeshSegNet/blob/c7dd252270e0558d0147b6fdfede03f85bfe354e/models/LitModule.py#L101

lovasz_loss = self.lovasz_fn(outputs.view(-1, self.cfg_model.num_classes), batch["label"].view(-1))

[error_loss.txt](https://github.com/ziyangyeh/iMeshSegNet/files/10907292/error_loss.txt)

**Comment by ziyangyeh** [comment]
Pull the newer submissions, then try again. I updated some codes to get better performance in iMeshSegNet.
If you have any deployment issue in iMeshSegNet, you may ask @koushiyu for further assistance, he/she did an amazing extension of deploying iMeshSeg in cpp.

**Comment by ziyangyeh** [comment]
The MeshSegNet is not recommended to use anymore. I did some experiments and updated the codebase, iMeshSegNet has a significant speedup and similar accuracy.
